### PR TITLE
Added eslint support

### PR DESCRIPTION
**Pull request by MounicaChanda0809**
Added eslint and prettier support which does a great job at handling code formatting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,9 +37,11 @@
   "scripts": {
     "start": "node dist/main.js",
     "build": "tsc --build .tsconfig.json",
-    "test": "jest --no-cache"
+    "test": "jest --no-cache",
+    "lint": "eslint '*/**/*.{js,ts,tsx}' --quiet --fix"
   },
   "dependencies": {
+    "eslint-plugin-react": "^7.21.3",
```
Comment (pyymenta) [inline package.json]: First of all, thank you for your contribution! ❤️

I have a question here: there is a reason to have a plug-in for react, considering our application is a CLI?

Comment (MounicaChanda0809) [reply package.json]: Ohh yeah, It was added accidentally. Will remove it.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  parser: "@typescript-eslint/parser", // Specifies the ESLint parser
+  parserOptions: {
+    ecmaVersion: 2020, // Allows for the parsing of modern ECMAScript features
+    sourceType: "module", // Allows for the use of imports
+    ecmaFeatures: {
+      jsx: true // Allows for the parsing of JSX
```
Comment (pyymenta) [inline .eslintrc.js]: I think we shouldn't have jsx support in this project too, it's not necessary, what do you think?

Comment (MounicaChanda0809) [reply .eslintrc.js]: Yeah true, Sorry I accidentally added react support too

**Comment by sudiptog81** [comment]
The builds for #39 shall pass after this is merged. Nothing to worry I guess.

**Comment by pyymenta** [comment]
> The builds for #39 shall pass after this is merged. Nothing to worry I guess.

Yeahh! You did it! Thank you so much! I'll merge your changes! :rocket:

**Pull request merged**